src/plugins_types.h

```diff
@@ -1199,12 +1201,12 @@ LIBYANG_API_DECL LY_ERR lyplg_type_validate_patterns(struct lysc_pattern **patte
  * @param[in] node Context node.
  * @param[in] value Target value.
  * @param[in] tree Full data tree to search in.
- * @param[out] target Optional found target.
+ * @param[out] target Optional set of found targets.
```

Comment:
I was trying to keep backward compatibility logic in the same/similar way as was the single pointer before. So the set is filled only if caller is needing it. The need is expressed by providing empty set as out parameter when it have non-NULL value.

Do you want to completely change this and rather always provide results to caller, even if caller is not interested in it?

Reply:
Currently, you can either pass `targets` `NULL` when the targets should not be returned or pass a pointer to an existing set to fill it with the targets. To follow all the other functions, I requested you change the latter functionality so that you pass pointer to a `NULL` set if you want the targets.

Reply:
Done